misc/userscripts/ripbang

```diff
@@ -20,15 +20,17 @@ except ImportError:
 for argument in sys.argv[1:]:
     bang = '!' + argument
     r = requests.get('https://duckduckgo.com/',
-                     params={'q': bang + ' SEARCHTEXT'})
+                     params={'q': bang + ' SEARCHTEXT'},
+                     headers={'user-agent': 'ripbang'})
```

Comment:
```suggestion
                     headers={'user-agent': 'qutebrowser ripbang'})
```

so that in the (unlikely) case there's something causing trouble, people looking at logs or so have an idea what "ripbang" is.